wiki/Legal/Copyright/de.md

```diff
@@ -0,0 +1,61 @@
+---
+legal: true
+---
+
+# Urheberrechtspolitik von osu!
+
+osu! ist ein stark an der Community orientiertes Spiel. Alle Spiellevel (Beatmaps) werden durch Community-Mitglieder erstellt und hochgeladen, oft ohne korrekte Einholung der Erlaubnis der jeweiligen Rechteinhaber. Wir tun unser Bestes, Nutzerinhalte nicht zu werben oder daraus Gewinn zu erzielen. Das Spiel wird vollständig aus Nutzerspenden finanziert.
+
+Ein Großteil der Spenden, die wir erhalten, werden für die [Musiklizenzierung](/wiki/Legal/Music_licensing) genutzt. Anfragen zur Lizenzvergabe können an [music@ppy.sh](mailto:music@ppy.sh) gesendet werden.
+
+In Ermangelung dessen hat osu! die folgende Leitlinie bezüglich Urheberrechtsverletzungen in Übereinstimmung mit dem Digital Millennium Copyright Act, 17 U.S.C. 512 verabschiedet.
+
+Die Kontaktinformationen des Beauftragten von osu! für die Entgegennahme von Mitteilungen über geltend gemachte Rechtsverletzungen (im folgenden Beauftragter genannt) wird am Ende dieser Leitlinie aufgelistet und ist beim United States Copyright Office (<https://www.copyright.gov/dmca-directory/>) hinterlegt. osu! nimmt Urheberrecht und andere Rechte an geistigem Eigentum sehr ernst.
+
+Es ist die Politik von osu!,
```

Comment:
I'd use "Richtlinie" (guideline) instead of "Politik" here and below. Politik sounds more like plan (like Klimapolitik), whereas Richtlinie sounds more like a set of rules on what to do after something has happened.

Since I'm not German myself, my feeling about these words might be wrong, so do with this what you want xD Maybe an English speaker can confirm that "policy" is used as "guideline" here.

Reply:
When learning about organisational structures in business economics, you often see this three-branch structure (with an example about baking bread):
- Leitlinie (`Policy`): bake bread (what do we do)
- Richtlinie (`Procedure`): bake the following bread, so that we are the best bakery (how to realise the policy)
- Anweisung (`Instruction`): take the following ingredients

I thought this would correspond with the legal text.

Reply:
Yeah nvm, "Politik" is probably the correct translation in the legal context